### ISSUE TITLE
Errors when compiling `json-validator.cpp` 

### ISSUE THREAD

**Issue by park53kr** [pboettch/json-schema-validator#41]
Hi. When I included `json-validator.cpp` file and built the project, the compiler gave some errors. 

## Error

```cpp
/Users/imac/enterprise-merger/include/json-validator.cpp:154:22: error: implicit instantiation of undefined template 'std::__1::basic_ostringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
                std::ostringstream s;
                                   ^
/Library/Developer/CommandLineTools/usr/include/c++/v1/iosfwd:123:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_ostringstream;
                               ^
/Users/imac/enterprise-merger/include/json-validator.cpp:177:21: error: implicit instantiation of undefined template 'std::__1::basic_ostringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
        std::ostringstream s;
                           ^
/Library/Developer/CommandLineTools/usr/include/c++/v1/iosfwd:123:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_ostringstream;
```

## Environment
- macos(Mojave 10.14.2), clang 7.0

I just added `#include <sstream>` and the errors were resolved.

**Comment by pboettch** [comment]
Could you create a pull-request for this issue?

**Comment by pboettch** [comment]
This issue is already solved via #39
